docs/data/base/components/switch/UnstyledSwitchIntroduction/tailwind/index.tsx

```diff
@@ -9,96 +8,49 @@ function useIsDarkMode() {
 }
 
 export default function UnstyledSwitchIntroduction() {
-  const label = { 'aria-label': 'Demo switch' };
-
   // Replace this with your app logic for determining dark modes
   const isDarkMode = useIsDarkMode();
 
   return (
     <div className={isDarkMode ? 'dark' : ''}>
-      <Switch slotProps={{ input: { ...label } }} defaultChecked />
-      <Switch slotProps={{ input: { ...label } }} />
-      <Switch slotProps={{ input: { ...label } }} defaultChecked disabled />
-      <Switch slotProps={{ input: { ...label } }} disabled />
+      <Switch aria-label="Demo switch" defaultChecked />
+      <Switch aria-label="Demo switch" />
+      <Switch aria-label="Demo switch" defaultChecked disabled />
+      <Switch aria-label="Demo switch" disabled />
     </div>
   );
 }
 
-const resolveSlotProps = (fn: any, args: any) =>
-  typeof fn === 'function' ? fn(args) : fn;
-
+interface SwitchProps {
+  'aria-label'?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  defaultChecked?: boolean;
+  readOnly?: boolean;
+}
 const Switch = React.forwardRef<HTMLSpanElement, SwitchProps>((props, ref) => {
+  const { 'aria-label': ariaLabel, ...other } = props;
   return (
     <BaseSwitch
       ref={ref}
-      {...props}
+      {...other}
       slotProps={{
-        ...props.slotProps,
-        root: (ownerState) => {
-          const resolvedSlotProps = resolveSlotProps(
-            props.slotProps?.root,
-            ownerState,
-          );
-          return {
-            ...resolvedSlotProps,
-            className: clsx(
-              `relative inline-block w-[38px] h-[24px] m-2.5  ${
-                ownerState.disabled
-                  ? 'cursor-not-allowed opacity-40'
-                  : 'cursor-pointer'
-              }`,
-              resolvedSlotProps?.className,
-            ),
-          };
+        root: {
+          className:
+            'relative inline-block w-[38px] h-[24px] m-2.5 ui-disabled:cursor-not-allowed ui-disabledopacity-40 cursor-pointer',
```

Comment:
This should be

```suggestion
            'relative inline-block w-[38px] h-[24px] m-2.5 ui-disabled:cursor-not-allowed ui-disabled:opacity-40 cursor-pointer',
```

But it's not working either 😅, `opacity-40` is not applied

Reply:
Fixed, now, looks like it works based on testing the docs. Let me know if you find that something is off.

Reply:
Seems to be working 🙌🏼